### ISSUE TITLE
[Mechanics] - use chamfer instead of bevel

### ISSUE THREAD

**Issue by Akustav** [ut-robotics/picr21-team-bts#7]
I suggest using chamfer here so certain parts are not printed on thin air:

![image](https://user-images.githubusercontent.com/5435954/135767231-1d7872bb-f2a3-4d31-8f9c-3cd1e728c837.png)

This change should not break anything on first check.

**Comment by mbz4** [comment]
Agreed, fillet angle exceeding 60degrees. Disregardable if using layer height under 0.15mm but will apply chamfer regardless.

**Comment by mbz4** [comment]
![image](https://user-images.githubusercontent.com/30321314/135767885-c754a4e4-42de-41bf-b5c5-3a557e5a47d8.png)
Done.
